test/test-upgrades.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+UPGRADEABLE_VERSIONS="
+    1.3.0
+    1.2.1
+    1.2.0
+"
+
+TEST_DATABASE=linz-bde-schema-upgrade-test-db
+
+export PGDATABASE=${TEST_DATABASE}
+
+for ver in ${UPGRADEABLE_VERSIONS}; do
+    OWD=$PWD
+
+    dropdb --if-exists ${TEST_DATABASE}
+    createdb ${TEST_DATABASE} || exit 1
+
+    psql -XtA <<EOF
+CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE SCHEMA IF NOT EXISTS _patches;
+CREATE EXTENSION IF NOT EXISTS dbpatch SCHEMA _patches;
+EOF
+
+    cd /tmp
+    wget -q -O - \
```

Comment:
* Isn't it better to git checkout to git tag than downloading tarball ?

Reply:
Better in terms of ? Bandwidth ? We may need a full clone for a git checkout, or at least a clone of as much data as needed to download all the tags we want to test ... I didn't profile it, would do what's simplest. This works, why change it ?

Reply:
Ok I understood we already have a clone of the git repo, so with new commit I've switched to using git instead of fetching the tarball